ntoskrnl/ob/oblink.c

```diff
@@ -442,7 +442,7 @@ ObpParseSymbolicLink(IN PVOID ParsedObject,
     POBJECT_SYMBOLIC_LINK SymlinkObject = (POBJECT_SYMBOLIC_LINK)ParsedObject;
     PUNICODE_STRING TargetPath;
     PWSTR NewTargetPath;
-    ULONG LengthUsed, MaximumLength, TempLength;
+    SIZE_T LengthUsed, MaximumLength, TempLength;
```

Comment:
Down below at lines 566,567, there is:
```c
        FullPath->MaximumLength = MaximumLength;
        FullPath->Length = LengthUsed;
```
Probably these lengths need to be cast to USHORT to avoid some warnings. (This is already done in lines 641, 642)

Reply:
They don't result in a warning. Either because MSVC is smart, or because MSVC is stupid.